lib/dynflow/testing/managed_clock.rb

```diff
@@ -26,6 +26,13 @@ def progress
         end
       end
 
+      def progress_all(ignore_list)
+        until @pending_pings.empty? || !ignore_list.include?(@pending_pings.first.what.try(:value))
```

Comment:
I'm sure there is some better way than calling `.try(:value)`, I initially was calling .value, but it hit an entry in @pending_pings that did not have that:  Algebrick::Types::None:Algebrick::Atom

Is there some better way to handle this condition?

Reply:
Well, you could always check the class of `@pending_pings.first` before checking if it is in the list, but its far from ideal. @iNecas do you recall if there's some algebrick trick for handling this situation?

Reply:
Not a better approach I can think of: also, long term I would like to get rid of algebrick for readability reasons, so I would rather avoid some of it's advanced features

Reply:
Just be aware that `try` does not come with pure ruby and we don't have runtime dependency on active support.

Reply:
oh right!  I'm spoiled living in a 🛤 world